docs/whatsnew/5.3.rst

```diff
@@ -111,32 +111,39 @@ It is now possible to read and write compressed FITS files that make use of the
 uncompressed tiles by specifying ``compression_type='NOCOMPRESS'`` in
 :class:`~astropy.io.fits.CompImageHDU`.
 
-.. _whatsnew-5.3-unit-formats-inline:
+.. _whatsnew-5.3-unit-formats-fraction-inline:
 
-New "inline" option for ``"unicode"``, ``"console"``, and ``"latex"`` unit formats
-==================================================================================
+New "fraction" option for representing units as strings
+=======================================================
+
+A new formatting option is added to switch between using fractions or
+using negative powers directly, with the fraction option also
+allowing to switch between inline and multiline prettyprinting of
+units::
 
-A new formatting option for unit is added to switch between inline and
-multiline prettyprinting of quantities::
 
 	>>> import astropy.units as u
 	>>> unit = u.Unit("erg / (s cm2)")
 	>>> print(unit.to_string('console'))
 	erg s^-1 cm^-2
-	>>> print(unit.to_string('console', inline=False))
+	>>> print(unit.to_string('console', fraction='inline'))
+        erg / (s cm^2)
+	>>> print(unit.to_string('console', fraction='display'))
          erg
         ------
         s cm^2
 	>>> print(unit.to_string('unicode'))
 	erg s⁻¹ cm⁻²
-	>>> print(unit.to_string('unicode', inline=False))
+	>>> print(unit.to_string('unicode', fraction='inline'))
+        erg / (s cm²)
+	>>> print(unit.to_string('unicode', fraction='display'))
          erg
         ─────
         s cm²
 
 Note that the ``"console"`` and ``"unicode"`` formats now use
-``inline=False`` as default, while for ``"latex"`` the default remains
-``inline=True``, for an unchanged experience with IPython notebook.
+``fraction=False`` by default, while for ``"latex"`` the default remains
+``fraction='display'``, for an unchanged experience with IPython notebook.
```

Comment:
Given the discussion over at https://github.com/astropy/astropy-APEs/pull/82, the whatsnew should probably include *why* we are making this change in default -- that this means display for `Quantity` is not too awful.

I'll do that once there is general agreement about this PR.

Reply:
I think, this *could* be fixed by fixing the unicode printing. However there are many places where one doesn't want to have multilines in strings; table headers and table cells are important use cases here. I'd go even so far to say that inline formats never break something (even when they may sometimes look not optimal), but multiline often breaks formatting and is optimal only specific cases. So, multiline should be clearly the way to go here.

Reply:
Good point about table headings! I'll add that too.

Reply:
Updated the whatsnew with an explanation.